### PR TITLE
Added fonts and  modified the font-size, margin, and  background-color

### PR DESCRIPTION
**Pull request by naoto-tomita**

### CODE REVIEW

File: src/Component/ApplicationBar.jsx [hunk 1]
```diff
@@ -1,25 +1,30 @@
 import React from 'react';
+import './applicationbar.scss';
 import { AppBar, Toolbar, Typography } from '@material-ui/core';
 import { Link } from 'react-scroll';
 
 const ApplicationBar = () => {
 
   return (
-    <div>
+    <div id = "header">
```
Comment (gomyway1216) [inline src/Component/ApplicationBar.jsx:9]: plz remove spaces between vals and use single quotes. ex) id='header'

Comment (gomyway1216) [reply src/Component/ApplicationBar.jsx:9]: also try to use className over id.

File: src/Component/ApplicationBar.jsx [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
           </IconButton> */}
-          <Typography variant="h6">
-            SmartBooks Inc.
-          </Typography>
-          <div style={{ width: '20%', display: 'flex', justifyContent: 'space-evenly', cursor: 'pointer'}}>
-            <Link activeClass="active" to='home' spy={true} smooth={true} duration={500}>Home</Link>
-            <Link activeClass="active" to='mission' spy={true} smooth={true} duration={500} >OUR MISSION</Link>
-            <Link to='team' spy={true} smooth={true} duration={500} >TEAM</Link>
-            <Link to='contact' spy={true} smooth={true} duration={500} >CONTACT</Link>
+          <div id = "logo">
```
Comment (gomyway1216) [inline src/Component/ApplicationBar.jsx:15]: same as above

**Review by gomyway1216 — COMMENTED** [review]
please fix the commented lines.

File: src/Component/ApplicationBar.jsx [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+            <Typography variant="h6">
```
Comment (gomyway1216) [inline src/Component/ApplicationBar.jsx:16]: same as above

**Comment by gomyway1216** [comment]
Also, there should be no changes for package-lock.json as you are not adding/updating dependencies. Please double check on that as well.

**Comment by naoto-tomita** [comment]
Noted. Thanks!

**Pull request closed without merging**